docs/embed-files.md

```diff
@@ -53,6 +53,17 @@ Example:
 
 [filename](_media/example.js ':include :type=code :fragment=demo')
 
+## Embedded file order
+
+Then the content of `delayed.md` `non-delayed.md` and will be displayed directly here in correct order
+
+[delayed](_media/delayed.md ':include')
+
+---
+
+[non-delayed](_media/non-delayed.md ':include')
+
+You can check the original content for [delayed.md](_media/delayed.md ':ignore'), [non-delayed.md](_media/non-delayed.md ':ignore').
 
 ## Tag attribute
 
```

Comment:
what we need is just telling users we will render embedded file in order by default. 
and I think it should do like this logically.

Reply:
reverted, block is inlined by cypress post fs sync